pkg/util/kubernetes/kubelet/podwatcher.go

```diff
@@ -21,6 +24,12 @@ type PodWatcher struct {
 	kubeUtil       *KubeUtil
 	expiryDuration time.Duration
 	lastSeen       map[string]time.Time
+	tagsDigests    map[string]digests
+}
+
+type digests struct {
```

Comment:
Is there a good reason for those hashes to be separate, or can we make a digest of both, get rid of the type, and make `tagsDigests` be a `map[string]string` instead?

Reply:
that's right, we can calculate a single digest of both labels and annotations and store it in a `map[string]string` 👍